nmostesting/suites/IS1101Test.py

```diff
@@ -13,14 +13,336 @@
 # limitations under the License.
 
 from ..GenericTest import GenericTest
+from .. import TestHelper
 
 COMPAT_API_KEY = "streamcompatibility"
+CONTROLS = "controls"
+NODE_API_KEY = "node"
+CONN_API_KEY = "connection"
 
 
 class IS1101Test(GenericTest):
     """
     Runs Node Tests covering IS-11
     """
+
     def __init__(self, apis):
-        GenericTest.__init__(self, apis)
+        # Don't auto-test paths responding with an EDID binary as they don't have a JSON Schema
+        omit_paths = [
+            "/inputs/{inputId}/edid",
+            "/inputs/{inputId}/edid/base",
+            "/inputs/{inputId}/edid/effective",
+            "/outputs/{outputId}/edid"
+        ]
```

Comment:
I agree with Nikita about using `omit_paths` as necessary for Connection API as well as Stream Compatibility API.

> I guess, it's planned to move tests, that access Node API and Connection API, to `IS1102Test.py`

Actually I have requested that because the test plan does not separate IS-11-only test cases from ones needing Node and/or Connection API, that all test cases should be combined into _IS1101Test.py_ for now.

> nmos-testing tries to find resources from the previously tested API in the current API

That sounds like a bug in _GenericTest.py_ if so? I think discovered resource ids during the auto-testing should only be used for child paths in the same API.

Reply:
In PR-783 we merged IS1101Test.py and IS1102Test.py. This discussion is therefore resolved. An other item should be open to track the GenericTest.py issue.

Reply:
I checked #783@23c6274, and `omit_paths` contain `/inputs/{inputId}/edid` for some reason and doesn't contain `/single/senders/{senderId}/transportfile`. As I understood, it modifies `GenericTest.py` and `TestHelper.py` instead that doesn't seem a good approach.

Reply:
Extracting https://github.com/AMWA-TV/nmos-testing/pull/754/commits/2cf3a525e65e160058c2b78bc4b8aaf7809220c8 to another PR makes sense to me. @garethsb, may I do it?

Reply:
Pulled out GenericTest.py and TestHelper.py and adding missing omit path